### PR TITLE
add GetThemeColor

### PR DESCRIPTION
**Pull request by jiuker**
add GetThemeColor
https://github.com/minio/mc/blob/67fa4aa7a3ed211f9b09a2b2901d5c3fac77ae2b/cmd/progress-bar.go#L38
When `mc` set `Bar` I can check if that set.

### CODE REVIEW

**Comment by shtripat** [comment]
Can you please explain the usecase in description a bit?

File: console/themes.go [hunk 1]
```diff
@@ -38,6 +38,14 @@ var (
 	themeMu sync.Mutex
 )
 
+// GetThemeColor gets a color for a particular tag.
+func GetThemeColor(tag string) (*color.Color, bool) {
```
Comment (vadmeste) [inline console/themes.go:42]: Can you use getThemeColor to avoid code duplication ?

**Review by vadmeste — CHANGES_REQUESTED** [review]

**Review by vadmeste — APPROVED** [review]
LGTM

**Review by shtripat — APPROVED** [review]

**Review by kannappanr — APPROVED** [review]

**Comment by jiuker** [comment]
No need any more.

**Pull request closed without merging**